### PR TITLE
Add Tests ruby 2.7 and 3.0 for integration test

### LINKED ISSUE #392: Tests for Ruby 2.7 and 3.0
Hi! Thank you for great gem.
Could you please add Ruby 3.0 and 2.7 tests as shown in the ranked-model commit below?
https://github.com/mixonic/ranked-model/pull/174
https://github.com/mixonic/ranked-model/pull/168

I didn't think it would be a good idea for me to just copy and paste this code and I will be a contributor, since it's mostly the result of @aried3r efforts!

### PR DESCRIPTION
**Pull request by QWYNG**
fix #392 

special thanks to @aried3r.
I thank him for the kind offer.

I referred to the following commit.
https://github.com/mixonic/ranked-model/pull/174
https://github.com/mixonic/ranked-model/pull/168

### CODE REVIEW

File: test/test_list.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def setup_db(position_options = {})
 
   # AR caches columns options like defaults etc. Clear them!
   ActiveRecord::Base.connection.create_table :mixins do |t|
-    t.column :pos, :integer, position_options unless position_options[:positive] && sqlite
+    t.column :pos, :integer, **position_options unless position_options[:positive] && sqlite
```
Comment (QWYNG) [inline test/test_list.rb:14]: Ruby 2.7.3 warning

```
❯ rake test                                                                                            
Run options: --seed 34339

# Running:

#{HOME}/acts_as_list/test/test_list.rb:14: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/#{HOME}/.rbenv/versions/2.7.3/lib/ruby/gems/2.7.0/gems/activerecord-6.1.3.1/lib/active_record/connection_adapters/abstract/schema_definitions.rb:401: warning: The called method `column' is defined here
..................................................................................................................................................................................................

Finished in 15.685896s, 12.3678 runs/s, 62.0940 assertions/s.

194 runs, 974 assertions, 0 failures, 0 errors, 0 skips
```

File: gemfiles/rails_5_0.gemfile [hunk 2]
```diff
@@ -17,11 +17,11 @@ group :test do
 end
 
 group :sqlite do
-  gem "sqlite3", "~> 1.3.13"
+  gem "sqlite3", "~> 1.4"
```
Comment (QWYNG) [inline gemfiles/rails_5_0.gemfile:20]: for 2.7 warning
https://github.com/sparklemotion/sqlite3-ruby/issues/276

File: .travis.yml [hunk 3]
```diff
@@ -26,7 +28,28 @@ gemfile:
...
+    - rvm: 2.4
       gemfile: gemfiles/rails_6_1.gemfile
+    # Ruby 2.7 uses a `bigdecimal` version that doesn't support BigDecimal.new
+    # that Rails 4.2 uses. See also:
+    # https://github.com/ruby/bigdecimal#which-version-should-you-select
+    - rvm: 2.7
+      gemfile: gemfiles/rails_4_2.gemfile
+    # Ruby 2.7 warning `sqlite3-1.3.13/lib/sqlite3/statement.rb:108: warning: rb_tainted_str_new is deprecated`
+    # and job wil exceed the maximum log length
+    # see also: https://github.com/sparklemotion/sqlite3-ruby/issues/276
```
Comment (QWYNG) [inline .travis.yml:42]: Example: https://travis-ci.org/github/brendon/acts_as_list/jobs/767571757

Comment (QWYNG) [reply .travis.yml:42]: The warning also occurs with the ranked-model.
https://travis-ci.org/github/mixonic/ranked-model/jobs/755121129

**Comment by brendon** [comment]
Looks good to me @QWYNG, well done on getting this all green. Travis can be a temperamental beast at times :D

**Pull request merged**